.gitignore

```diff
@@ -31,29 +31,29 @@ docs/man/man3/*
 # docsite stuff...
 docs/api/_build/
 docs/api/rst/
-docs/docsite/*.html
 docs/docsite/_build
-docs/docsite/_static/*.gif
-docs/docsite/_static/*.png
-docs/docsite/_static/websupport.js
+docs/docsite/*.html
 docs/docsite/htmlout
-docs/docsite/searchindex.js
-docs/docsite/rst_warnings
-docs/docsite/rst/*_module.rst
-docs/docsite/rst/ansible.rst
-docs/docsite/rst/ansible-*.rst
-docs/docsite/rst/community_maintained.rst
-docs/docsite/rst/config.rst
-docs/docsite/rst/core_maintained.rst
-docs/docsite/rst/list_of_*.rst
-docs/docsite/rst/module_docs/*.rst
-docs/docsite/rst/modules_by_category.rst
-docs/docsite/rst/network_maintained.rst
-docs/docsite/rst/plugins_by_category.rst
-docs/docsite/rst/partner_maintained.rst
-docs/docsite/rst/playbooks_keywords.rst
+docs/docsite/rst/cli/ansible-*.rst
+docs/docsite/rst/cli/ansible.rst
+docs/docsite/rst/modules/community_maintained.rst
```

Comment:
Isn't `modules/*.rst` sufficient over all these other specific .rsts in the same dir?